### PR TITLE
feat: Update minimal embed mode chart footnotes

### LINKED ISSUE #962: Change Source Label for Embedded Charts
BAFU has asked to refine the label shown in the "source" section below the embedded charts

Current: Weitere Informationen
Image

Expected: /visualize.admin.ch 

Image

### PR DESCRIPTION
**Pull request by bprusinowski**
Closes #962.

### CODE REVIEW

File: app/components/chart-footnotes.tsx [hunk 1]
```diff
@@ -263,3 +257,18 @@ export const ChartFootnotes = ({
     return null;
   }
 };
+
+const LinkButton = (props: PropsWithChildren<{ href: string }>) => {
```
Comment (ptbrowne) [inline app/components/chart-footnotes.tsx:261]: TIL PropsWithChildren !

File: app/components/chart-footnotes.tsx [hunk 2]
```diff
@@ -174,6 +174,11 @@ export const ChartFootnotes = ({
                 dangerouslySetInnerHTML={{ __html: dataCubeByIri.publisher }}
               ></Box>
             )}
+            {configKey && shareUrl && visualizeLinkText && (
+              <>
+                / <LinkButton href={shareUrl}> {visualizeLinkText}</LinkButton>
+              </>
+            )}
```
Comment (ptbrowne) [inline app/components/chart-footnotes.tsx:181]: Wondering if there will not be a space missing before the slash ?

**Review by ptbrowne — APPROVED** [review]
Just wonder about the space before the slash not being rendered, otherwise LGTM

File: app/components/chart-footnotes.tsx (hunk 2 above)
Comment (bprusinowski) [reply app/components/chart-footnotes.tsx:182]: Good catch, I missed that!

**Pull request merged**